src/components/pages/Analyze.tsx

```diff
@@ -0,0 +1,32 @@
+
+import React, { useContext } from "react";
+import { AppContext } from "context";
+import { ANALYZE_URLS, PAGE_HASHES } from "../../constants";
+import TableauEmbed from "components/shared/TableauEmbed";
+import "./static.css";
+
+export default function Analyze() {
+
+  const [{ language }] = useContext(AppContext);
+
+  return (
+    <div className="page col-12">
+      <TableauEmbed
+        url={ANALYZE_URLS}
+        key={`AnalyzeTableau${language}`}
+        desktopOptions={{
```

Comment:
where are these `desktopOptions` and `mobileOptions` used?

Reply:
they are passed to the tableau component to size the graphs properly